frontend/src/components/v-summary-charts.vue

```diff
@@ -316,6 +330,24 @@ export default {
       return repo.location;
     },
 
+    getRepoType(repo) {
+      const repoLink = this.getRepoLink(repo);
+
+      if (repoLink.includes('github')) {
+        return 'github';
+      }
+
+      if (repoLink.includes('gitlab')) {
+        return 'gitlab';
+      }
+
+      if (repoLink.includes('bitbucket')) {
+        return 'bitbucket';
+      }
+
+      return 'default';
```

Comment:
Is it possible to declare `'default'` as some kind of constant variable that can be referenced here and in lines 45 and 115.

Reply:
Sure, let me google and update it

Reply:
Regarding this, I now have a better idea. Instead of returning the type, I can straight away return the icon. Trying to work towards this goal now.

Reply:
Is it fine to leave the 'github', 'gitlab', and 'bitbucket' as they are?

Reply:
yep, those are fine